web/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I'm struggling with prettier's preference for double quoting everything. It's been 10y of single quotes and I'm finding the the change jarring. 
You don't mind the double quotes?

Reply:
I think with the nx changes the whole repo will share the same .prettierrc file (unless explicitly overridden), which may end up modifying these files back to the single quotes (oops, wasn't trying to create extra churn but as the login component PR is based on this one it's probably still better to merge?).